### DISCUSSION TITLE
An error when I build examples using cmake

### DISCUSSION THREAD

**Discussion by Monsterever** [AMReX-Codes/amrex#3921] [Q&A] [answered]
There is an error when I build examples using cmake.

CMake Error at CMakeLists.txt:12 (setup_tutorial):
  Unknown CMake command "setup_tutorial".

I check the CMakeLists and find that the function called setup_tutorial is not highlighted.
The version of cmake is 3.22.1, Ubuntu20.04.
Should I get a higher version of cmake?

**Comment by WeiqunZhang** [comment]
How are you building it? Could you provide more details?

It seems to work for me. (I assume you are talking about amrex-tutorials. There is no `setup_tutorial` in the amrex repo.)
```
$ cd amrex-tutorials/ExampleCodes
$ git log -n 1
commit e2f1008b7fa43a95d490fa2833146639541cfa98 (HEAD -> main, origin/main, origin/HEAD)
Author: Weiqun Zhang <weiqunzhang@lbl.gov>
Date:   Thu Apr 11 13:31:32 2024 -0700

    Update make for SUNDIALS
$ mkdir build
$ cd build
$ cmake ..
$ make -j8
```

**Reply by Monsterever** [reply]
Yes I used the same steps to build the tutorial, but is shown in the figure below.
![image](https://github.com/AMReX-Codes/amrex/assets/68806975/c9938fa7-df6a-4b6a-bcae-af270b12d529)

**Reply by WeiqunZhang** [reply, marked as answer]
`cd amrex-tutorials/ExampleCodes`, not `amrex-tutorials/ExampleCodes/Basic/HeatEquation_EX1_C`. That's how the build system is set up. Then the executable for HeatEquation_EX1_C will appear at `amrex-tutorials/ExampleCodes/build/Basic/HeatEquation_EX1_C`.

**Reply by Monsterever** [reply]
Excellent! Thank you!
